src/OrchardCore.Modules/OrchardCore.ReverseProxy/Drivers/ReverseProxySettingsDisplayDriver.cs

```diff
@@ -15,63 +16,56 @@ namespace OrchardCore.ReverseProxy.Drivers
 {
     public class ReverseProxySettingsDisplayDriver : SectionDisplayDriver<ISite, ReverseProxySettings>
     {
-        private const string RestartPendingCacheKey = "ReverseProxySiteSettings_RestartPending";
         private const string SettingsGroupId = "ReverseProxy";
-
-        private readonly INotifier _notifier;
-        private readonly IMemoryCache _memoryCache;
+        private readonly IShellHost _orchardHost;
+        private readonly ShellSettings _currentShellSettings;
 
         public ReverseProxySettingsDisplayDriver(INotifier notifier,
-            IMemoryCache memoryCache,
+            IShellHost orchardHost,
+            ShellSettings currentShellSettings,
             IHtmlLocalizer<ReverseProxySettingsDisplayDriver> stringLocalizer)
         {
-            _notifier = notifier;
-            _memoryCache = memoryCache;
+            _orchardHost = orchardHost;
+            _currentShellSettings = currentShellSettings;
             T = stringLocalizer;
         }
         IHtmlLocalizer T { get; }
 
-        public override IDisplayResult Edit(ReverseProxySettings settings, BuildEditorContext context)
+        public override IDisplayResult Edit(ReverseProxySettings section, BuildEditorContext context)
         {
```

Comment:
`settings` is used everywhere else

Reply:
Then we should change the interface, my implementation adheres to the interface ;)

Reply:
As you wish. Probably something we should clean up, but another pr later.
About 80% use `settings` and the others `section`

Other than that all LGTM